### ISSUE TITLE
How to dinamically disable a button on the Toolbar

### ISSUE THREAD

**Issue by fpuga** [geoman-io/leaflet-geoman#642]
Is there any recommend way to dinamically disable a button (or a mode) in the Toolbar.

I have an entity that can have or not a polygon geometry. This geometry will never be a multipolygon. If it has a geometry it is the only editable layer in the map.

I want all the available buttons visible in the map

```javascript
map.pm.addControls({
    position: "topleft",
    drawMarker: false,
    drawCircleMarker: false,
    drawPolyline: false,
    drawRectangle: true,
    drawPolygon: true,
    drawCircle: true,
    editMode: true,
    dragMode: true,
    cutPolygon: false,
    removalMode: true,
});
```

But enable/disable while still visible, after user actions.

- If the entity already has a geometry, or the user draw one, only edit/drag/remove mode buttons should be "clickable"
- If the entity does not have a geometry, or the user removes it, only draw mode buttons should be "clickable"

AFAIK, i can modify the css getting the buttons with

```javascript
map.pm.Toolbar.getButtons().drawPolygon._button
// or 
map.pm.Toolbar.getButtons().drawPolygon.buttonsDomNode
// or
document.getElementsByClassName('leaflet-pm-draw')
```

but i'm not sure about the correct approach to handle the "click" action. Maybe it could be a new feature directly supported by Geoman.

Currently what i'm doing is something like

```javascript

// disable buttons
document.getElementsByClassName('leaflet-pm-draw')[0].style.pointerEvents = 'none';
document.querySelectorAll('.leaflet-pm-draw a').forEach(a => a.classList.add('leaflet-disabled'))

//enable buttons
document.getElementsByClassName('leaflet-pm-draw')[0].style.pointerEvents = 'auto';
document.querySelectorAll('.leaflet-pm-draw a').forEach(a => a.classList.remove('leaflet-disabled'))
```

**Comment by Falke-Design** [comment]
@fpuga Thanks for reporting. This is already on my TODO list also I will close this Issue and add it to the [feature requests](https://github.com/geoman-io/leaflet-geoman/issues?q=is%3Aissue+is%3Aclosed+label%3A%22feature+request%22+sort%3Areactions-%2B1-desc) where users can upvote this request.

**Comment by Falke-Design** [comment]
This is released with the Version 2.8.0. Call `map.pm.Toolbar.setButtonDisabled(name, Boolean)`
